src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/test_vm_commands.py

```diff
@@ -1504,7 +1506,7 @@ def test_vmss_existing_lb(self, resource_group):
         vmss_name = 'vmss1'
         lb_name = 'lb1'
         self.cmd('network lb create -g {} -n {} --backend-pool-name test'.format(resource_group, lb_name))
-        self.cmd('vmss create -g {} -n {} --load-balancer {} --image UbuntuLTS --admin-password TestTest12#$'.format(resource_group, vmss_name, lb_name))
+        self.cmd('vmss create -g {} -n {} --load-balancer {} --image UbuntuLTS --admin-username clitester --admin-password TestTest12#$'.format(resource_group, vmss_name, lb_name))
```

Comment:
unrelated, but a simple fix so the test can run in my mac with a log-on'd user name which fails CLI's admin name check